javascript-source/games/gambling.js

```diff
@@ -115,7 +115,7 @@
 		 * @commandpath gamblesetgainpercent [amount in percent] - Set the winning gain percent.
 		 */
 		if (command.equalsIgnoreCase('gamblesetgainpercent')) {
-			if (!action) {
+			if ((!action) || isNaN(action) || (action < 1) || (action > 100)) {
```

Comment:
You don't need to add everything in specific parentheses, also since you're updating this change `!action` to `action === undefined`, and use `isNaN(parseInt(action))` something rhino will mess that up.

e.g `((!action) || (action < 1))`